index.css

```diff
@@ -0,0 +1,49 @@
+div {
+    background-color: lightblue;
+  }
+
+  p {
+    background-color: yellow;
+  }
+
+  li {
+    font-size:16pt;
+}
+
+ul li{
+    font-size:25px;
+    font-family: Arial, Sans-Serif;
```

Comment:
"Sans-Serif" (capitalized) is a non-standard font family, which may be recognized by some browsers (i.e. they process the value is if it were all lower-case), but may NOT be recognized by other browsers.  Consider using lower-case for all font family values unless you are using a particular font which IS capitalized (rare).  See line 21 below for an example of a "sans-serif" all lower-case, which is the standard name for that font family.